getDataFRF.py

```diff
@@ -2619,3 +2630,40 @@ def getLidarWaveProf(self, removeMasked=True):
             print 'There is no LIDAR data during this time period'
             out = None
         return out
+
+    def getCSHOREOutput(self, prefix):
+        """retrives data from spatial data STWAVE model
+            
+        Args:
+            prefix (str): a 'key' to select which version of the simulations to pull data from
+                available value is only 'MOBILE_RESET' for now but more could be 
+                added in the future
+                
```

Comment:
please define output dictionary (similar to above functions)

Reply:
Done